### PR TITLE
Added example for pixelated image rendering.

### PR DESCRIPTION
**Pull request by pearlchen-old**
For feature: http://www.chromestatus.com/feature/5118058116939776

Preview: http://pearlchen.github.io/samples/image-rendering-pixelated/

### CODE REVIEW

File: any-pointer-any-hover/index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    <title>'any-pointer' and 'any-hover' Media Queries Sample</title>
+
+    <link href='//fonts.googleapis.com/css?family=Roboto+Condensed:300italic,400italic,700italic,400,300,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="../styles/main.css">
+  </head>
+  <body>
+    <h1>'any-pointer' and 'any-hover' Media Queries Sample</h1>
+    <p>Available in <a href="http://www.chromestatus.com/feature/6460705494532096">Chrome 38+</a></p>
+
+    <p>The <code>any-pointer</code> and <code>any-hover</code> give web authors the ability to feature detect </p>
```
Comment (jeffposnick) [inline any-pointer-any-hover/index.html]: This looks like an incomplete sentence.

File: any-pointer-any-hover/index.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+      <dt>hover</dt>
+      <dd></dd>
+
+    </dl>
+
+    <!-- // [START code-block] -->
+    <style>
+
+      /* Make radio buttons and check boxes larger if we have an inaccurate pointing device */
```
Comment (jeffposnick) [inline any-pointer-any-hover/index.html]: Could you add some `<input type="checkbox">` and `<div class="menu>` elements to the page to illustrate how the styles influence the way the elements are rendered?

**Comment by pearlchen-old** [comment]
@jeffposnick Sorry, I didn't mean to actually submit a PR, just an issue. Here's my question from the spreadsheet:

"Here's a draft sample code. I feel like it'll be more appropriate to break them into 2 separate samples/files though. Logistically, what do you think the best approach is? 2 separate directories? Or keep in the same place?"

**Comment by jeffposnick** [comment]
A separate directory for each distinct feature fits with the overall model we have going right now.

If you could split out the pixelated demo (which I think is good to go) from the any-pointer/hover demo (which probably still needs work) and just include the pixelated demo in this PR that would be ideal. Creating separate git branches for each sample should make that easier.

**Comment by pearlchen-old** [comment]
Oh, I see what happened. I pushed the wrong local branch to my gh-pages. It's back to just the pixelated image rendering example.

**Pull request merged**